### ISSUE TITLE
NTR, pulmo: lack of surfactant-protein B (SP-B)

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5565]
**Preferred term label:**
lack of dimeric surfactant-protein B (SP-B)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Significantly decreased level or failed detection of surfactant protein B in broncho-alveolar lavage fluid. Comment: Pulmonary surfactant is a highly surface-active mixture of proteins and lipids that is synthesized and secreted onto the alveoli by type II epithelial cells. The protein part of surfactant constitutes of four types of surfactant proteins (SP), SP-A, SP-B, SP-C and SP-D. SP-A and SP-D are hydrophilic proteins that regulate surfactant metabolism and have immunologic functions, whereas SP-B and SP-C are hydrophobic molecules, which play a direct role in the organization of the surfactant structure in the interphase and in the stabilization of the lipid layers during the respiratory cycle. Lack of SP-B may result of surfactant metabolism dysfunction. 

PMID: 26375475

**Parent term (use hpo.jax.org/app)**
Abnormal biochemical composition of broncho-alveolar fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 265120, OMIM:  267450 

**Your nano-attribution (ORCID)**
crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
Label -- Absent dimeric surfactant-protein B in bronchoalveolar lavage

**Comment by pnrobinson** [comment]
adding as HP:0032981
